dashboard/app/models/levels/blockly.rb

```diff
@@ -226,6 +226,14 @@ def self.convert_category_to_toolbox(xml_string)
     xml.serialize(save_with: XML_OPTIONS).delete("\n").strip
   end
 
+  # "counter" mutations should not be stored because it results in the language being hardcoded
```

Comment:
Can you elaborate in this comment that the mutation *should* be stored for student code, it just shouldn't be stored in the level configuration XML (start/toolbox/solution blocks)

Reply:
Yes, definitely should have as much clarity as possible here.